lib/peep.rb

```diff
@@ -0,0 +1,24 @@
+class Peep
+  attr_reader :content, :date
+
+  def initialize(id: nil, content: nil, date: nil)
```

Comment:
There probably shouldn't be a scenario where you create a Peep with any of these values as nil, as they are all required (and should be NOT NULL in the database).